### PR TITLE
NitroHistory to SimpleTrail gem

### PR DESCRIPTION
**Pull request by ghaydarov**
This PR adds SimpleTrail gem to PowerTools.

### CODE REVIEW

File: packages/.DS_Store
Comment (gbmoretti) [inline packages/.DS_Store:1]: I think you can .gitignore that

File: packages/simple_trail/.rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: AllowedMethods.
+# AllowedMethods: enums
+Lint/ConstantDefinitionInBlock:
+  Exclude:
+    - 'spec/models/simple_trail/history_spec.rb'
```
Comment (that-jill) [inline packages/simple_trail/.rubocop_todo.yml:14]: can you try to address these violations? our goal with new packages is to not need the todo file at all -- it's mentioned in the documentation on power-tools, and these two may be pretty easy to resolve.

File: packages/simple_trail/Gemfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# Declare your gem's dependencies in simple_trail.gemspec.
+# Bundler will treat runtime dependencies like base dependencies, and
+# development dependencies will be added by default to the :development group.
+gemspec
+
+rails_version = ENV.fetch("RAILS_VERSION", ">= 5")
```
Comment (that-jill) [inline packages/simple_trail/Gemfile:10]: it looks like we are only testing Rails 6 versions, so we should probably specify that here.

File: packages/simple_trail/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install simple_trail
+
+## Usage
```
Comment (that-jill) [inline packages/simple_trail/docs/README.md:22]: is there a need to include `simple_trail` in the load_path anywhere?

File: packages/simple_trail/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Now you can access the object history through `SimpleTrail.for(object)` like:
+
+```ruby
+project = Project.create
+SimpleTrail.for(project).size
+# => 1
+```
+
+The user performing the action will be recorded from the Thread local `:user_id`.
```
Comment (that-jill) [inline packages/simple_trail/docs/README.md:38]: I'm not sure what use case this is here. Will a user_id be recorded on a table? Where? How? What if I want something other than a user_id?

File: packages/simple_trail/spec/dummy/config/webpacker.yml
Comment (xjunior) [inline packages/simple_trail/spec/dummy/config/webpacker.yml:1]: Remove this

File: packages/simple_trail/spec/dummy/log/development.log
Comment (xjunior) [inline packages/simple_trail/spec/dummy/log/development.log:1]: Remove this file

File: packages/simple_trail/spec/dummy/public/packs-test
Comment (xjunior) [inline packages/simple_trail/spec/dummy/public/packs-test:1]: Remove this file

File: packages/simple_trail/.rubocop_todo.yml (hunk 1 above)
Comment (ghaydarov) [reply packages/simple_trail/.rubocop_todo.yml:14]: I did fix one of the issues that was causing the `rubocop` issue, but there is still one. I will come back to this later.

File: docs/README.md [hunk 5]
```diff
@@ -46,6 +46,10 @@ A shared layout so that your suite of applications can have the same look and fe
 
 Edgestitch allows engines to define partial structure-self.sql files to be stitched into a single structure.sql file by the umbrella application.
 
+[simple_trail](https://github.com/powerhome/power-tools/blob/main/packages/simple_trail/docs/README.md) 💎
+
+SimpleTrail makes it easy to keep history of attribute changes on a model.
```
[the remark this answers is not in the document]
Comment (that-jill) [reply docs/README.md:51]: bumping ^^^

File: packages/simple_trail/.gitignore [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+spec/dummy/db/simple_trail_test.sqlite
+spec/dummy/db/simple_trail_development.sqlite
+db/simple_trail_test.sqlite
+db/simple_trail_development.sqlite
+spec/dummy/log/test.log
+spec/dummy/log/development.log
+spec/dummy/db/structure.sql
+spec/dummy/public/packs-test
+spec/dummy/config/webpacker.yml
+.gitkeep
```
[the remark this answers is not in the document]
Comment (ghaydarov) [reply packages/simple_trail/.gitignore:26]: Sounds good. I will remove it from here @xjunior .

**Review by xjunior — APPROVED** [review]

**Review by xjunior — APPROVED** [review]

**Pull request merged**